code/parachain/runtime/picasso/src/weights/pablo.rs

```diff
@@ -0,0 +1,91 @@
+//! HOSTNAME: `20dcb8f4ced6`, CPU: `Intel(R) Xeon(R) CPU @ 2.20GHz`
+//! EXECUTION: Some(Wasm), WASM-EXECUTION: Compiled, CHAIN: Some("dali-dev"), DB CACHE: 1024
+
+// Executed Command:
+// ./target/release/composable
+// benchmark
+// pallet
+// --chain=dali-dev
+// --execution=wasm
+// --wasm-execution=compiled
+// --wasm-instantiation-strategy=legacy-instance-reuse
+// --pallet=*
+// --extrinsic=*
+// --steps=50
+// --repeat=20
+// --output=parachain/runtime/dali/src/weights
+// --log
+// error
+
+#![cfg_attr(rustfmt, rustfmt_skip)]
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+
+use frame_support::{traits::Get, weights::Weight};
+use sp_std::marker::PhantomData;
+
+/// Weight functions for `pablo`.
+pub struct WeightInfo<T>(PhantomData<T>);
+impl<T: frame_system::Config> pablo::WeightInfo for WeightInfo<T> {
+	// Storage: CurrencyFactory AssetIdRanges (r:1 w:1)
+	// Storage: Pablo PoolCount (r:1 w:1)
+	// Storage: StakingRewards RewardPools (r:1 w:1)
+	// Storage: Timestamp Now (r:1 w:0)
+	// Storage: Fnft Collection (r:1 w:1)
+	// Storage: CurrencyFactory AssetEd (r:0 w:3)
+	// Storage: Pablo Pools (r:0 w:1)
+	fn create() -> Weight {
+		(161_782_000 as Weight)
+			.saturating_add(T::DbWeight::get().reads(5 as Weight))
+			.saturating_add(T::DbWeight::get().writes(8 as Weight))
+	}
+	// Storage: Pablo Pools (r:1 w:0)
+	// Storage: Tokens Accounts (r:5 w:5)
+	// Storage: Tokens TotalIssuance (r:1 w:1)
+	// Storage: System Account (r:1 w:1)
+	// Storage: Pablo PriceCumulativeState (r:1 w:1)
+	fn add_liquidity() -> Weight {
+		(245_338_000 as Weight)
+			.saturating_add(T::DbWeight::get().reads(9 as Weight))
+			.saturating_add(T::DbWeight::get().writes(8 as Weight))
+	}
+	// Storage: Pablo Pools (r:1 w:0)
+	// Storage: Tokens Accounts (r:5 w:5)
+	// Storage: Tokens TotalIssuance (r:1 w:1)
+	// Storage: System Account (r:1 w:0)
+	// Storage: Pablo PriceCumulativeState (r:1 w:1)
+	fn remove_liquidity() -> Weight {
+		(223_464_000 as Weight)
+			.saturating_add(T::DbWeight::get().reads(9 as Weight))
+			.saturating_add(T::DbWeight::get().writes(7 as Weight))
+	}
+	// Storage: Pablo Pools (r:1 w:0)
+	// Storage: Tokens Accounts (r:4 w:4)
+	// Storage: System Account (r:2 w:1)
+	// Storage: Pablo PriceCumulativeState (r:1 w:1)
+	fn buy() -> Weight {
+		(183_216_000 as Weight)
+			.saturating_add(T::DbWeight::get().reads(8 as Weight))
+			.saturating_add(T::DbWeight::get().writes(6 as Weight))
+	}
+	// Storage: Pablo Pools (r:1 w:0)
+	// Storage: Tokens Accounts (r:5 w:5)
+	// Storage: System Account (r:3 w:2)
+	// Storage: StakingRewards RewardPools (r:1 w:1)
+	// Storage: Timestamp Now (r:1 w:0)
+	// Storage: Pablo PriceCumulativeState (r:1 w:1)
+	fn swap() -> Weight {
+		(285_540_000 as Weight)
+			.saturating_add(T::DbWeight::get().reads(12 as Weight))
+			.saturating_add(T::DbWeight::get().writes(9 as Weight))
+	}
+
+	fn do_create_pool() -> Weight {
+		10_000
+    }
```

Comment:
We'll want to make sure we generate weights for this PR

Reply:
We will do this for the release as per the [RFC](https://github.com/ComposableFi/composable/blob/main/rfcs/0011-releases.md#43-act) sir